src/DynamoCoreWpf/ViewModels/Search/SelectionNavigator.cs

```diff
@@ -70,6 +75,26 @@ internal void MoveSelection(NavigationDirection direction)
             if (root == null || !root.Any())
                 return;
 
+            if (selection == topResult)
+            {
+                // We can only move forward...
+                if (direction == NavigationDirection.Forward)
+                {
+                    selectedCategoryIndex = 0;
+                    selectedMemberGroupIndex = 0;
+                    selectedMemberIndex = 0;
+
+                    topResult.IsSelected = false;
+                    selection = GetSelectionFromIndices();
+                    selection.IsSelected = true;
```

Comment:
Can we change `topResult.IsSelected = false` to `selection.IsSelected = false`? I'd like to keep `topResult` used as limited as possible, and the rest are all dealing with `selection` (so that it becomes clear that we are doing something with the *current selection*).